flexmeasures/data/models/reporting/tests/test_tibber_reporter.py

```diff
@@ -130,7 +138,11 @@ def __init__(self, sensor) -> None:
             final_df_output="da_prices",
         )
 
-        super().__init__(sensor, reporter_config)
+        self._inner_reporter = PandasReporter(config=pandas_reporter_config)
+
+    def _compute_report(self, **kwargs):
+        kwargs["input_sensors"] = self.input_sensors
+        return self._inner_reporter.compute(**kwargs)
```

Comment:
Why can't we subclass from `PandasReporter` anymore?

Reply:
In this case, both ways work as we don't require a custom config/input for the `TibberReporter`. If it was the case, we would require schemas for the inner (`PandasReporter`) and the outer  `TibberReporter`, and manage the deserialization. Nonetheless, I chose to change to serve as an example on how to create Reporters that use other Reporters.